src/entities/Admin.ts

```diff
@@ -0,0 +1,28 @@
+import { Entity, PrimaryGeneratedColumn, Column, Unique } from 'typeorm';
+
+@Entity('admins')
+@Unique(['email'])
+export class Admin {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  email: string;
+
+  @Column()
+  password: string;
```

Comment:
Como a senha vai ser guardada encriptada, poderíamos criar um [Listener](https://typeorm.io/listeners-and-subscribers#entity-listeners-and-subscribers) com o **TypeORM** para que sempre que um usuário for criado, a sua senha também seja gerada e encriptada, automatizando esse processo.

Algo nessa linha:
```ts
@BeforeInsert()
public async encryptPassword(): Promise<void> {
  if (this.password) {
     this.password = generateAndHashPassword();
  }
}
```